src/components/Molecules/List/FancyTweetList/FancyTweetList.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import {
+  Card,
+  CardActionArea,
+  CardContent,
+  Typography,
+  CardActions,
+  Button,
+} from '@material-ui/core';
+
+interface IFancyTweetListProps {
+  tweets: any[];
+}
+
+const styles = {
+  card: {
+    maxWidth: 345,
+  },
+  media: {
+    height: 140,
+  },
+};
+
+const FancyTweetList = ({ tweets }: IFancyTweetListProps) => {
```

Comment:
What especially makes this `TweetList` 'fancy'? 😉

Reply:
`TweetList` also fine. Renamed.